scripts/obico.sh

```diff
@@ -0,0 +1,422 @@
+      echo -e "| install more Moonraker-obico instances than available |"
+      echo -e "| Moonraker instances.                                  |"
+      bottom_border
+
+      ### ask for amount of instances
+      local new_moonraker_obico_count re="^[1-9][0-9]*$"
+      while [[ ! ${new_moonraker_obico_count} =~ ${re} || ${new_moonraker_obico_count} -gt ${allowed_moonraker_obico_count} ]]; do
+        read -p "${cyan}###### Number of new Moonraker-obico instances to set up:${white} " -i "${allowed_moonraker_obico_count}" -e new_moonraker_obico_count
+        ### break if input is valid
+        [[ ${new_moonraker_obico_count} =~ ${re} && ${new_moonraker_obico_count} -le ${allowed_moonraker_obico_count} ]] && break
+        ### conditional error messages
+        [[ ! ${new_moonraker_obico_count} =~ ${re} ]] && error_msg "Input not a number"
+        (( new_moonraker_obico_count > allowed_moonraker_obico_count )) && error_msg "Number of Moonraker-obico instances larger than installed Moonraker instances"
+      done && select_msg "${new_moonraker_obico_count}"
+    else
+      log_error "Internal error. new_moonraker_obico_count of '${new_moonraker_obico_count}' not equal or grather than one!"
+      return 1
+    fi
+
+    ### Step 2: Confirm instance amount
+    local yn
+    while true; do
+      (( new_moonraker_obico_count == 1 )) && local question="Install Moonraker-obico?"
+      (( new_moonraker_obico_count > 1 )) && local question="Install ${new_moonraker_obico_count} Moonraker-obico instances?"
+      read -p "${cyan}###### ${question} (Y/n):${white} " yn
+      case "${yn}" in
+        Y|y|Yes|yes|"")
+          select_msg "Yes"
+          break;;
+        N|n|No|no)
+          select_msg "No"
+          abort_msg "Exiting Moonraker-obico setup ...\n"
+          return;;
+        *)
+          error_msg "Invalid Input!";;
+      esac
+    done
+  fi
+
+  if (( new_moonraker_obico_count > 0 )); then
+
+    ### Step 3: Ask for the Obico server URL
+    obico_server_url_prompt
+    local obico_server_url
+    while true; do
+      read -p "${cyan}###### Obico Server URL:${white} " -i "https://app.obico.io" -e obico_server_url
+      if echo "${obico_server_url}" | grep -qE "^(http|https)://[a-zA-Z0-9./?=_%:-]*"; then
+        break
+      else
+        error_msg "Invalid server URL!"
+      fi
+    done
+
+    (( new_moonraker_obico_count > 1 )) && status_msg "Installing ${new_moonraker_obico_count} Moonraker-obico instances ..."
+    (( new_moonraker_obico_count == 1 )) && status_msg "Installing Moonraker-obico ..."
+
+    ### Step 4: Install dependencies
+    local dep=(git dfu-util virtualenv python3 python3-pip python3-venv ffmpeg)
+    dependency_check "${dep[@]}"
+
+    ### Step 5: Clone the moonraker-obico repo
+    clone_or_update_moonraker_obico "${MOONRAKER_OBICO_REPO}"
+
+    ### step 6: call moonrake-obico/install.sh with the correct params
+    local log="${KLIPPER_LOGS}"
```

Comment:
```ssh
local log="${KLIPPER_LOGS}"
```
This variable seems to be unused?

Reply:
Another good catch. I guess I planed to use it but ended up using `"${KLIPPER_LOGS}"` directly. Feel free to remove it.